src/pageLayout/utils/index.test.ts

```diff
@@ -71,20 +71,5 @@ describe('pageLayout.utils.getNavItemActivation', () => {
     expect(getNavItemActivation([given], `${base}${given}/${given}`)).toEqual(
       true
     )
-
-    given = 'telegrafs'
-    expect(getNavItemActivation([given], `${base}/${given}${given}`)).toEqual(
-      false
-    )
-
-    given = 'scrapers'
-    expect(
-      getNavItemActivation([given], `${base}/${given}${given}${given}`)
-    ).toEqual(false)
-
-    given = 'tokens'
-    expect(getNavItemActivation([given], `${base}////////${given}`)).toEqual(
-      true
-    )
```

Comment:
removed these because we now want this `getNavItemActivation` to handle `/`s in the keywords. And these cases aren't valid parths anyways